pkg/common/utils/utils.go

```diff
@@ -50,85 +46,15 @@ const (
 var getLatestCRDVersion = kubernetes.GetLatestCRDVersion
 
 // ListVirtualMachines lists all the virtual machines
-// converted to the latest API version(v1alpha4).
-// Since, VM Operator converts all the older API versions to the latest version,
-// this function determines the latest API version of the VirtualMachine CRD and lists the resources.
 func ListVirtualMachines(ctx context.Context, clt client.Client,
-	namespace string) (*vmoperatorv1alpha5.VirtualMachineList, error) {
+	namespace string) (*vmoperatorv1alpha2.VirtualMachineList, error) {
```

Comment:
On a general note: This is clearly for pvcsi VM operations. Why not simply create a folder called pvcsi, and move all this to a file called vm_utils.go. This way, we need not worry about ever misusing the function. You can do this in a different follow-up MR.